File: src/main/java/dev/kikugie/elytratrims/common/access/BannerLayer.java
package dev.kikugie.elytratrims.common.access;

import net.minecraft.block.entity.BannerPattern;
import net.minecraft.registry.entry.RegistryEntry;
import net.minecraft.util.DyeColor;

public record BannerLayer(RegistryEntry<BannerPattern> pattern, DyeColor color) {
}

File: src/main/kotlin/dev/kikugie/elytratrims/common/access/BannerLayer.kt
package dev.kikugie.elytratrims.common.access

import net.minecraft.block.entity.BannerPattern
import net.minecraft.registry.entry.RegistryEntry
import net.minecraft.util.DyeColor

data class BannerLayer(val pattern: RegistryEntry<BannerPattern>, val color: DyeColor)
